profile/change_password.php

```diff
@@ -16,6 +16,16 @@
 
 global $_current_user;
 
+
+    /**
+	 * Verify that a string is Sha_1
+	 * @access  public
+	 * @param   $str : Sha_1 Encryted String
+	 */
+function is_sha1($str)
```

Comment:
Instead of repeatedly defining this function at various locations, can you define it in a shared script such as [Utility.class.php](https://github.com/inclusive-design/AChecker/blob/master/include/classes/Utility.class.php)? Thanks.